### ISSUE TITLE
CDN email address is undeliverable

### ISSUE THREAD

**Issue by SamirTalwar** [FortAwesome/Font-Awesome#19498]
The email address, cdn@fontawesome.com, on the help page, https://cdn.fontawesome.com/help, is broken. Email is undeliverable.

Either the email needs to be resurrected or changed to a working email address.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

That is an old website with an old project

Please refer to fontawesome.com. Email address is hello@fontawesome.com

/cc @robmadole

**Comment by SamirTalwar** [comment]
Fair enough. I suggest at least putting a banner or something on the site then, so it's clear it's old.

**Comment by tagliala** [comment]
> Fair enough. I suggest at least putting a banner or something on the site then, so it's clear it's old.

Good suggestion, I agree
